### PR TITLE
Api-Validations Complete

### PR DESCRIPTION
**Pull request by monicaarganda**

### CODE REVIEW

File: spec/models/hero_spec.rb [hunk 1]
```diff
@@ -1,5 +1,92 @@
...
+  #     post '/heros', params: hero_params
+  #     # expect an error if the hero_params does not have a name
+  #     expect(response.status).to eq 422
+  #     # Convert the JSON response into a Ruby Hash
+  #     json = JSON.parse(response.body)
+  #     # Errors are returned as an array because there could be more than one, if there are more than one validation failures on an attribute.
+  #     expect(json['name']).to include "can't be blank"
+  #   end
+  # end
+# end
```
Comment (SunkissedQueen) [inline spec/models/hero_spec.rb]: Please remove code that you will not be using or provide an explanation to keep it

**Review by SunkissedQueen — CHANGES_REQUESTED** [review]
Aloha, I really like how you have made the trello reflect your project. Good job! So the change request is addressing the commented code on `spec/models/hero_spec.rb` for the stretch goal. In the field, the reviews are made to look over code that we want to present to the client. Unless there is an explanation to keep it, please remove it. I can help with blockers outside of the review. During the review, consider me a client. And right now as a client I do not want any code in the main branch of the application that is not working.

File: spec/models/hero_spec.rb (hunk 1 above)
Comment (monicaarganda) [reply spec/models/hero_spec.rb]: Thank you for the feedback! I will remove the code.

**Review by SunkissedQueen — APPROVED** [review]
Thanks. Your client is happy

**Pull request merged**